plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeConfig.java

```diff
@@ -84,6 +84,7 @@ public class DeltaLakeConfig
     private boolean registerTableProcedureEnabled;
     private boolean projectionPushdownEnabled = true;
     private boolean queryPartitionFilterRequired;
+    private boolean deletionVectorsEnabled;
```

Comment:
What's the default for this in delta/databricks ?

Reply:
OSS Delta Lake disables it by default. Databricks depends on the version (it's disabled at version < 14.x).

Reply:
If latest Databricks enables it by default, we should work towards enabling this by default as well in future